crud.py

```diff
@@ -0,0 +1,75 @@
+from sqlalchemy.orm import Session
+import models
+import schemas
+from fastapi import HTTPException
+
+
+def get_all_authors(
+        db: Session,
+        skip: int | None = None,
```

Comment:
The type hint `int | None` is not compatible with Python versions below 3.10. If you are targeting an older version, consider using `Optional[int]` from the `typing` module instead.

Reply:
i use python 3.12